docs/learning/backend/PullMerge.md

```diff
@@ -0,0 +1,154 @@
+# Pull merge & conflict resolution
```

Comment:
This is a very informative article. I see no links to it - how are people supposed to find it? Please add some links from appropriate API docs.

Reply:
I think this article need to be referenced from https://www.itwinjs.org/learning/backend/concurrencycontrol/#optimistic-concurrency-control and that doc has to be updated as its content also now out dated.

Reply:
Ok, let's leave it for a follow-up PR.